lib/mocha.js

```diff
@@ -883,3 +932,33 @@ Mocha.prototype.run = function(fn) {
 
   return runner.run(done);
 };
+
+/**
+ * Runs root suite and invokes `fn()` when complete. Supports ES Modules.
+ *
+ * @description
+ * To run tests multiple times (or to run tests in files that are
+ * already in the `require` cache), make sure to clear them from
+ * the cache first!
+ *
```

Comment:
- pls add that ES modules can't be cleared from the cache.
- _Mocha.prototype.unloadFiles_: aprox line380, add to description that ES modules can't be deleted from cache.

These description will be printed in our API docs.